### PR TITLE
Created a new index page

### PR DESCRIPTION
**Pull request by ReditusDraco**
Noticed there was no index page so created a new one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world.
```
Comment (ecarlste) [inline index.html]: Chris I'd really like to see an exclamation point instead! Please update the code before resubmitting. Cheers mate!

**Review by ecarlste — CHANGES_REQUESTED** [review]
Please see comments and change as appropriate.

File: index.html (hunk 1 above)
Comment (ReditusDraco) [reply index.html]: I have completed your request sir!

Comment (ecarlste) [reply index.html]: you sir, are, the best! SHIP IT!!!

**Review by ecarlste — APPROVED** [review]
Great work! Looks good!

**Pull request merged**